README.md

```diff
@@ -1,40 +1,92 @@
+instances of this object in the database:
+
+```scala
+// First declare our Scala object
+final case class Coffee(name: String, price: Double)
+
+// Next define how Slick maps from a database table to Scala objects
+class Coffees(tag: Tag) extends Table[Coffee](tag, "COFFEES") {
+  def name  = column[String]("NAME")
+  def price = column[Double]("PRICE")
+  def * = (name, price).mapTo[Coffee]
+}
+
+// The `TableQuery` object gives us access to Slick's rich query API
+val coffees = TableQuery[Coffees]
+
+// Inserting is done by appending to our query object
+// as if it were a regular Scala collection
+// SQL: insert into COFFEES (NAME, PRICE) values ('Latte', 2.50)
+coffees += Coffee("Latte", 2.50)
+
+// Fetching data is also done using the query object
+// SQL: select NAME from COFFEES
+coffees.map(_.name)
+
+// More complex queries can be chained together
+// SQL: select NAME, PRICE from COFFEES where PRICE < 10.0 order by NAME
+coffees.filter(_.price < 10.0).sortBy(_.name)
+```
+
+## Database support
+
+The following databases are directly supported by Slick, and are currently covered
+by a large suite of automated tests to ensure compatibility:
 
 |Database|JDBC Driver|
 |--------|-----------|
-|SQLServer 2008, 2012, 2014, 2017|[jtds:1.3.1](http://sourceforge.net/projects/jtds/files/jtds/) and [msjdbc:7.2.2](https://docs.microsoft.com/en-us/sql/connect/jdbc/download-microsoft-jdbc-driver-for-sql-server?view=sql-server-2017)|
-|Oracle 11g|[ojdbc7:12.1.0.2](http://www.oracle.com/technetwork/database/features/jdbc/index-091264.html)|
 |DB2 10.5|[db2jcc4:4.19.20](http://www-01.ibm.com/support/docview.wss?uid=swg21363866)|
+|Derby/JavaDB|derby:10.14.2.0|
+|H2|com.h2database.h2:1.4.199|
+|HSQLDB/HyperSQL|hsqldb:2.4.1|
 |MySQL|mysql-connector-java:8.0.16|
+|Oracle 11g|[ojdbc7:12.1.0.2](http://www.oracle.com/technetwork/database/features/jdbc/index-091264.html)|
 |PostgreSQL|postgresql:42.2.5|
 |SQLite|sqlite-jdbc:3.27.2.1|
-|Derby/JavaDB|derby:10.14.2.0|
-|HSQLDB/HyperSQL|hsqldb:2.4.1|
-|H2|com.h2database.h2:1.4.199|
-
-Accessing other database systems is possible, with a reduced feature set.
+|SQLServer 2008, 2012, 2014, 2017|[jtds:1.3.1](http://sourceforge.net/projects/jtds/files/jtds/) and [msjdbc:7.2.2](https://docs.microsoft.com/en-us/sql/connect/jdbc/download-microsoft-jdbc-driver-for-sql-server?view=sql-server-2017)|
 
-The [manual and scaladocs](https://scala-slick.org/docs/) for Slick can be
-found on the [Slick web site](https://scala-slick.org).
+Accessing other database systems is possible, although possibly with a reduced feature 
+set.
 
-## Maintenance status
+## Contributing
 
-Slick is community-maintained by a loose assortment of volunteers.
-Please help if you can.
+Slick is maintained by the Scala community. Pull requests are very welcome, and we
+ask that all contributors abide by the [Lightbend Community Code of Conduct](https://www.lightbend.com/conduct).
```

Comment:
Is this a new policy?

Reply:
No: it's always been the case from when Lightbend used to officially maintain slick (the code of conduct is  mentioned [on the website](https://scala-slick.org/community/) for example).

I thought it was worth making it explicit here - although we might want to consider switching to e.g. the more general Scala COC in future? (But that's a job for another day)

Reply:
fwiw, I'm certain Lightbend wouldn't object if you changed it to Scala CoC.